cmd/clusterawsadm/cmd/alpha/bootstrap/bootstrap.go

```diff
@@ -142,6 +143,60 @@ func createStackCmd() *cobra.Command {
 	return newCmd
 }
 
+func generateJSON() *cobra.Command {
+	newCmd := &cobra.Command{
+		Use:   "generate-bootstrap-json [Directory for JSON]",
```

Comment:
`generate-bootstrap-json` seems a bit redundant in this context, since the full command is `clusterawsadm alpha bootstrap generate-bootstrap-json`, what about using `generate-cloudformation-template` instead?

Reply:
Landed on `generate-iam-policy-docs`. WDYT?